src/examples/table/TableStripedRows.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Body, Cell, Head, HeaderCell, HeaderRow, Row, Table } from '@zendeskgarden/react-tables';
+
+interface IRowData {
+  index: number;
+  fruit: string;
+  sun: string;
+  soil: string;
+}
+
+const rowData: IRowData[] = [];
+
+for (let x = 0; x < 10; x++) {
+  rowData.push({
+    index: x,
+    fruit: 'Strawberries',
+    sun: 'Full sun',
+    soil: 'Well draining'
+  });
+}
```

Comment:
Similar to the comment above regarding imperative vs declarative, I think an `[].from()` with `[].map()` can be used to improve the example code.